src/test/java/ScattTest.java

```diff
@@ -25,10 +25,11 @@ public void testHello() {
     @Test
     public void testGui() {
```

Comment:
As an additional note i think this pull request should be closed and ignored. The current GUI work is ahead of this and actually functioning. I think I was hasty in putting in this request.